apps/condo/domains/miniapp/schema/SendAppPushMessageService.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Generated by `createservice miniapp.SendAppPushMessageService --type mutations`
+ */
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, FORBIDDEN } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+
+const { WRONG_FORMAT, DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const access = require('@condo/domains/miniapp/access/SendAppPushMessageService')
+const { MessageAppBlackList } = require('@condo/domains/miniapp/utils/serverSchema')
+const { B2CApp } = require('@condo/domains/miniapp/utils/serverSchema')
+const { VOIP_INCOMING_CALL_MESSAGE_TYPE, B2C_APP_MESSAGE_PUSH_TYPE } = require('@condo/domains/notification/constants/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { User } = require('@condo/domains/user/utils/serverSchema')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+const { USER_NOT_FOUND_ERROR, APP_NOT_FOUND_ERROR, APP_BLACK_LIST_ERROR } = require('../constants')
+
+const CACHE_TTL = {
+    DEFAULT: 3600,
+    VOIP_INCOMING_CALL_MESSAGE: 2,
+    B2C_APP_MESSAGE_PUSH: 3600,
+}
+
+const redisGuard = new RedisGuard()
```

Comment:
It's better to use existing redisGuard if possible. But it's may be not a part of your PR. 
Every RedisGuard produce a new redis connection. And we have 10 containers it means +10 connections.